### DISCUSSION TITLE
DELLA mini split AC remote

### DISCUSSION THREAD

**Discussion by robcloot** [Arduino-IRremote/Arduino-IRremote#1197] [Help]
I want to turn on and off my DELLA mini split AC with an ESP32 module. I successfully created a program using the IRremote library to control my TCL TV. When I used the following code, as recommended by the example code, it does not work. I even verified that my ESP32 remote was sending the correct code. Has anyone successfully duplicated a DELLA remote?

Send with:
    uint64_t tRawData[]={0x982040000226CB23, 0xDD00000000C0};
    IrSender.sendPulseDistanceWidthFromArray(38, 3000, 1650, 500, 1100, 500, 300, &tRawData[0], 112, PROTOCOL_IS_LSB_FIRST, 100, 0);

**Comment by ArminJo** [comment]
Check the number of repeats

**Reply by robcloot** [reply]
I changed the repeats to 0, 1, and 2 with no luck.
